frontend/src/app/modules/main/main-home/main-home.component.html

```diff
@@ -83,6 +83,9 @@
             <app-artist-card *ngIf="entity.entityType === 2"
               [artist]="entity.entity">
             </app-artist-card>
+            <app-artist-card *ngIf="entity.entityType === 3"
+              [artist]="entity.entity">
+            </app-artist-card>
```

Comment:
I think you should combine entityType === 2 and entityType === 3 into one if statement